app/test_main.py

```diff
@@ -1,3 +1,51 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (-1, -1, [0, 0])
+    ],
+    ids=[
+        "(0, 0) cat/dog years should be converted to [0, 0]",
+        "(14, 14) cat/dog years should be converted to [0, 0]",
+        "(15, 15) cat/dog years should be converted to [1, 1]",
+        "(23, 23) cat/dog years should be converted to [1, 1]",
+        "(24, 24) cat/dog years should be converted to [2, 2]",
+        "(27, 27) cat/dog years should be converted to [2, 2]",
+        "(28, 28) cat/dog years should be converted to [3, 2]",
+        "(100, 100) cat/dog years should be converted to [21, 17]",
+        "Invalid input should return [0, 0]",
```

Comment:
I'd prefer to see messages not as a standalone list, but as a bonus parameter, but it's up to you.
Example:
```python
@pytest.mark.parametrize(
    "cat_years, dog_years, expected, test_id",
    [
        (0, 0, [0, 0], "0 cat/dog should convert into [0, 0]"),
    ]
)
def test_get_human_age(cat_years, dog_years, expected, test_id):
    result = get_human_age(cat_years, dog_years)
    assert result == expected, test_id
```